lab-zach/.gitignore

```diff
@@ -0,0 +1,86 @@
+
```

Comment:
Make sure to include node_modules in your .gitignore so they aren't pushed up to github!